src/config/app/domain.rs

```diff
@@ -57,15 +50,9 @@ static RESERVED_KEYWORDS: [&str; 63] = [
     "throw",
     "throws",
     "transient",
-    "true",
```

Comment:
I just tried to build a Tauri project with identifier `fun.true.api` and it failed saying `true` is a Java keyword.. maybe we need to revert most of this

```
Package 'fun.true.api' from AndroidManifest.xml is not a valid Java package name as 'true' is a Java keyword
```

Reply:
`"true, false, and null might seem like keywords, but they are actually literals; you cannot use them as identifiers in your programs."` okay.. I missed this part